validator/src/main/kotlin/br/com/guiabolso/schemas/SchemaRepositoryFactory.kt

```diff
@@ -0,0 +1,18 @@
+package br.com.guiabolso.schemas
+
+import br.com.guiabolso.config.SchemaStorageType.FILESYSTEM
+import br.com.guiabolso.config.SchemaStorageType.S3
+import br.com.guiabolso.config.s3BucketName
+import br.com.guiabolso.config.storageType
+import com.amazonaws.regions.Regions
+
+object SchemaRepositoryFactory {
+
+    val schemaRepository: SchemaRepository = when (storageType()) {
```

Comment:
Acho que como é uma LIB não é necessário ter essa factory..

Sugestão: Deixaria o SchemaRepository como parametro do Validator e quem for usar instancia da maneira que preferir